packages/streamline/src/primitives/markdown/markdown.tsx

```diff
@@ -81,7 +81,7 @@ const styles = StyleSheet.create({
   link: {
     ...fonts.body,
     color: colors.BLUKO.BLUKO_500,
-    paddingBottom: isIOS ? 2 : 0,
+    paddingBottom: isIOS ? 1.2 : -1.5,
```

Comment:
You could export it into `markdown.constants.ts` like this
```tsx
const LINK_BOTTOM_SPACE = isIOS ? 1.2 : -1.5,
```

Ans use it in both `Markdown` and `MarkdownLink`.